robotd/devices_base.py

```diff
@@ -11,12 +11,15 @@ class BoardMeta(type):
     """
 
     BOARDS = []
+    STARTUP_BOARDS = []
 
     def __new__(cls, name, bases, dict):
         """Instantiate subclass."""
         cls = type.__new__(cls, name, bases, dict)
         if hasattr(cls, 'lookup_keys') and cls.enabled:
             BoardMeta.BOARDS.append(cls)
+        if hasattr(cls, 'create_on_startup'):
```

Comment:
This should probably check for `enabled`—and I think `create_on_startup` should probably be defined in the base class since it's a flag so that we can just do `if cls.create_on_startup`. Also avoids the slightly surprising result that `create_on_startup = False` still results in the board going into `STARTUP_BOARDS`!